packages/stark-ui/src/modules/pagination/components/pagination.component.ts

```diff
@@ -70,7 +70,7 @@ export class StarkPaginationComponent extends MatPaginator implements OnInit, On
 	 * StarkPaginationConfig object containing main information for the pagination.
 	 */
 	@Input()
-	public paginationConfig: StarkPaginationConfig;
+	public paginationConfig: StarkPaginationConfig = { totalItems: 0 };
```

Comment:
This variable can just be initialized as `{}` cause we already set the `totalItems = 0` in the `normalizePaginationConfig()` method ;)

So we just set the value in one place